src/utils/helpers.ts

```diff
@@ -44,7 +44,7 @@ export const handleNumber = (value, onlyPositive = true) => {
   }
 
   if (value.indexOf('.') === -1) {
-    value = value.repalce(',', '.');
+    // value = value.repalce(',', '.');
```

Comment:
uncomment it again, and change the typo, repalce -> replace.

Reply:
fixed with replace. haha